config-overrides.js

```diff
@@ -0,0 +1,59 @@
+const path = require('path');
+
+/**
+ * Configures `react-app-rewired` with the necessary changes to make the app work in the browser
+ * environment. This is necessary since the upgrade to WebPack 5.
+ * @param config
+ * @returns {*}
+ */
+module.exports = function override(config) {
+  // Resolves the modules that are missing in the browser environment
+  config.resolve.fallback = {
+    ...config.resolve.fallback,
+    buffer: require.resolve('buffer'),
+    crypto: require.resolve('crypto-browserify'),
+    stream: require.resolve('stream-browserify'),
+    path: require.resolve('path-browserify'),
+    vm: require.resolve('vm-browserify'),
+    url: require.resolve('url'),
+    assert: require.resolve('assert'),
+    process: require.resolve('process/browser'),
```

Comment:
Hey, some of these libraries are very old, check the wallet-desktop polyfills: https://github.com/HathorNetwork/hathor-wallet/blob/7e4734aa14275d529d9345e60a179d5418097cb8/config-overrides.js#L18-L29

Maybe we use the same library I used there?

Reply:
Thank for the suggestion! Implemented this library on https://github.com/HathorNetwork/hathor-explorer/pull/386/commits/8c134fe527663d53654ad33bb37c60e2db4f968c